timescaledb/how-to-guides/toolkit/counter_agg.md

```diff
@@ -0,0 +1,164 @@
+# Counter aggregates
+There are two types of metrics that can be recorded in a database: gauges, and
+counters. Gauges fluctuate up and down, like temperature or speed, while
```

Comment:
There are far more types than that and I wouldn't want to imply that these are the only two types of metrics one can store in a database I think we need to figure out how to limit this to the sort of constrained "metrics in the sense that Prometheus and other APM type things talk about". This feels misleading.